.gitignore

```diff
@@ -14,6 +14,7 @@ cmake_install.cmake
 /docs/generated/
 /coremltools.egg-info/
 /dist/
+/envs/
```

Comment:
Just curious, why is this line needed?

Reply:
The wheel build process creates environments under the `/envs/` directory:

https://github.com/apple/coremltools/blob/master/scripts/env_create.sh#L13

So I wanted to make sure git ignored that side effect of the build process.